Source/interfaces/IDisplayInfo.h

```diff
@@ -70,6 +111,11 @@ namespace Exchange {
         virtual uint32_t VerticalFreq() const = 0;
         virtual HDRType Type() const = 0;
         virtual HDCPProtectionType HDCPProtection() const = 0;
+        virtual int GetHdmiPreferences() const = 0;
+        virtual void SetHdmiPreferences(const int HdcpProtocol) = 0;
+        virtual IHdrIterator* GetTvHdrCapabilities() const = 0;
+        virtual IHdrIterator* GetStbHdrCapabilities() const = 0;
+        virtual bool IsOutputHDR() const = 0;
```

Comment:
@pwielders, Based on your suggestion at https://github.com/rdkcentral/Thunder/pull/214#discussion_r467864644 ,
I have moved the HDR and HDMI related methods to the Connection Interface. Could you please confirm these changes are aligned with  your expectations